options.go

```diff
@@ -132,7 +131,7 @@ type Options struct {
 	Filesystem billy.Filesystem
 	// These options are specifically used when envbuilder is invoked as part of a
 	// Coder workspace.
-	CoderAgentURL *url.URL
+	CoderAgentURL string
```

Comment:
Should we add this in?

```suggestion
	// Revert to `*url.URL` once https://github.com/coder/serpent/issues/14 is fixed.
	CoderAgentURL string
```